src/variants/api/ymodem.rs

```diff
@@ -1,4 +1,19 @@
-use crate::common::ModemTrait;
+use core::str::from_utf8;
+use core2::io::*;
+use core::alloc;
+
+use crate::common::*;
+#[cfg(defmt)]
+use defmt::*;
+use heapless::{String, Vec};
+
+const SOH: u8 = 0x01;
+const STX: u8 = 0x02;
+const EOT: u8 = 0x04;
+const ACK: u8 = 0x06;
+const NAK: u8 = 0x15;
+const CAN: u8 = 0x18;
+const CRC: u8 = 0x43;
```

Comment:
Why not use `crate::ymodem::Consts`? That was written specifically to avoid single-line consts. Please adjust.